app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,81 @@
+require 'twilio-ruby'
+require 'dotenv'
+
+class TweetsController < ApplicationController
+  get '/tweets' do
+    if !Helpers.is_logged_in?(session)
+      redirect to '/login'
+    else
+      @user = User.find_by_id(session[:user_id])
+      @tweets = Tweet.all
+      erb :'/tweets/index'
+    end
+  end
+
+  get '/tweets/new' do
+    if Helpers.is_logged_in?(session)
+      erb :'/tweets/new'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if params[:content].empty?
```

Comment:
refactored to validate presence of tweet content...not 100% that I caught all instances, but this one, at least, is updated